imagescan/controller.go

```diff
@@ -433,3 +450,20 @@ func (s *Controller) syncFromRemoteState(ctx context.Context) {
 	}
 	s.log.Infof("images updated from remote state, full_resync=%v, scanned_images=%d", resp.Images.FullResourcesResyncRequired, len(resp.Images.ScannedImages))
 }
+
+func isImagePending(v *image, now time.Time) bool {
+	if v == nil {
```

Comment:
These checks looks redundant. Also, if you move these helpers to image itself you don't need this checks.

```
func (img *image) isPending(now time.Time) bool {

}

func (img *image) isPrivate() bool {

}
```